addons/web/static/src/core/utils/scrolling.js

```diff
@@ -46,3 +48,13 @@ export function scrollTo(element, options = { scrollable: null, isAnchor: false
         }
     }
 }
+
+export function keepScrollTopPositionOnPatch(nodeRef) {
+    let previousScrollTop = 0;
+    onWillPatch(() => {
+        previousScrollTop = nodeRef.el.scrollTop;
+    });
+    onPatched(() => {
+        nodeRef.el.scrollTop = previousScrollTop;
+    });
+}
```

Comment:
maybe a bit overkill to have a function for this (which is a hook actually). I think I would do that directly in kanban_renderer (having the logic defined close to where it is used makes it easier to understand), and if someday we need this somewhere else, we'll generalize it.

Reply:
done